ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/hibernate/search/mapper/orm/main/module.xml

```diff
@@ -35,8 +35,10 @@
         <module name="jakarta.enterprise.api"/>
         <module name="jakarta.transaction.api"/>
         <module name="org.jboss.logging" />
+        <module name="org.jboss.jandex"/>
         <module name="org.hibernate.search.engine" export="true" />
         <module name="org.hibernate.search.mapper.pojo" export="true" />
         <module name="org.hibernate" />
+        <module name="org.hibernate.search.jipijapa-hibernatesearch" services="import" />
```

Comment:
This comment is just a note for future reference about provisioning. tl;dr; it's all good.

In a slimmed server the org.hibernate.search.mapper.orm module that's being modified here is only provisioned if the hibernate-search layer is provisioned. And this new org.hibernate.search.jipijapa-hibernatesearch is only provisioned because of this dependency. So this new module is handled consistently with the others; it only gets provisioned in an unslimmed server or if the layer is used.